modules/nf-core/cowpy/meta.yml

```diff
@@ -0,0 +1,48 @@
+name: "cowpy"
+description: Print any text in a cow or other characters
+keywords:
+  - cowpy
+  - hello
+  - ascii_art
+tools:
+  - "cowpy":
+      description: "Print any text in a cow or other characters"
+      homepage: "https://github.com/jeffbuttars/cowpy"
+      documentation: "https://github.com/jeffbuttars/cowpy"
+      tool_dev_url: "https://github.com/jeffbuttars/cowpy"
+      licence: ["Apache-2.0"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test', single_end:false ]
+    - text:
+        type: file
+        description: text file
+        pattern: "*.{txt}"
+output:
+  - txt:
+      - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+          pattern: "*.txt"
+      - ${prefix}.txt:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
```

Comment:
are you sure about this?

Reply:
you mean the example or the meta map?

Reply:
If it's the example, I don't really care, it's an example.
If it's the meta map, I do want one, so yeah I'm fairly sure